pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,356 @@
+- `description`: A short description for the tag.
+- `externalDocs`: Additional external documentation for this tag.
+
+### `Tags` in `Info` Object
+When defined within the `tags` property of the info object, tags provide a high-level categorization and classification for the entire AsyncAPI document. These tags that are defined under the `info` object give a global context to the entire application, representing overarching themes or functional areas within the event-driven system. They serve as a way to group objects such as channels or servers based on their broader significance, providing a holistic understanding of the application's structure.
+
+Here's a visual representation of the `tags` object inside an `info` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[asyncapi] --> B[info]
+  B --> C[title]
+  B --> D[version]
+  B --> E[description]
+  B --> F[tags]
+  F --> G[name]
+  F --> I[description]
+  F --> K[externalDocs]
+
+  style A fill:#47BCEE,stroke:#000;
+  style B fill:#47BCEE,stroke:#000;
+  style F fill:#47BCEE,stroke:#000;
+  style I fill:#47BCEE,stroke:#000;
+  style K fill:#47BCEE,stroke:#000;
+  style G fill:#47BCEE,stroke:#000;
+```
+
+Below is an example of the `tags` object inside the `info` object in an AsyncAPI document:
+```yaml
+asyncapi: 3.0.0
+info:
+  title: AsyncAPI Documentation
+  version: 1.0.0
+  description: |
+    This AsyncAPI document provides an overview
+    of the event-driven system.
+  tags:
+    - name: Applications
+      description: All applications related topics. 
+      url: https://applications.example.com/docs
+    - name: Time
+      description: All time related topics.
+      url: https://time.example.com/docs
+```
+
+### `Tags` in `Servers` Object
+Tags when used within the `tags` property of the `servers` object, are specific to the servers' configurations and relate to the server-level characteristics. These tags allow for the categorization of server instances based on their properties or criteria, such as geographical location, environment (e.g., production, development), or specific server capabilities. Using `tags` in the `servers` object allows for the categorization and organization of servers based on specific tags or labels. Using the `tags` object under the `servers` object is optional.
+
+Here's a visual representation of the `tags` object inside a `servers` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[asyncapi] --> B[servers]
+  B --> C[host]
+  B --> D[description]
+  B --> E[protocol]
+  B --> F[protocolVersion]
+  B --> G[tags]
+  G --> H[name]
+  G --> I[description]
+  G --> J[externalDocs]
+  style A fill:#47BCEE,stroke:#000;
+  style B fill:#47BCEE,stroke:#000;
+  style G fill:#47BCEE,stroke:#000;
+  style I fill:#47BCEE,stroke:#000;
+  style H fill:#47BCEE,stroke:#000;
+  style J fill:#47BCEE,stroke:#000;
+```
+
+Below is an example of the `tags` object inside the `servers` object in an AsyncAPI document:
+```yaml
+servers:
```

Comment:
I think it is good to have copy paste examples, like you have in first paragraph, so maybe better do

```yml
asyncapi: 3.0.0

info:
  title: AsyncAPI Documentation
  version: 1.0.0

servers:
  development:
    host: localhost:5672
    description: Development AMQP broker.
    protocol: amqp
    protocolVersion: 0-9-1
    tags:
      - name: "env:development"
        description: "This environment is meant for developers to run their tests."
  production:
    host: rabbitmq.in.mycompany.com:5672
    description: RabbitMQ broker for the production environment.
    protocol: amqp
    protocolVersion: 0-9-1
    tags:
      - name: "env:production"
        description: "This environment is the live environment available for final users."
```

Reply:
Addressed.